apps/i18n/netsim/en_us.json

```diff
@@ -2,6 +2,8 @@
   "a_and_b": "A/B",
   "addPacket": "Add Packet",
   "ascii": "ASCII",
+  "autoDnsNotFound": "NOT_FOUND",
```

Comment:
Do we really want this translated? If so, NOT_FOUND doesn't seem like all that friendly a string.